cypress/e2e/chris.cy.ts

```diff
@@ -0,0 +1,39 @@
+const xml2js = require('xml2js')
+
+describe("", () => {
+    it("Obstacle #72946", () => {
+        cy.visit('https://obstaclecourse.tricentis.com/Obstacles/72946');
+        // download the xml file
+        cy.request({
+            url: '/Obstacles/GetObstacleFile?obstacleId=72946&filename=Catalog.xml',
+            method: 'GET',
+            encoding: 'binary',
+          }).then((response) => {
+            const fileName = 'Catalog.xml';
+            cy.writeFile(fileName, response.body, 'binary');
+          });
+        // convert xml to json
```

Comment:
Why not use xpath to retrieve the node value?

Reply:
not applicable any more